action.yml

```diff
@@ -59,6 +64,10 @@ runs:
           echo "Invalid input for endtoend_config"
           exit 1
         fi
+        if ! [[ "$ENDTOEND_TAGS" =~ ^[a-zA-Z0-9]*$ ]]; then
```

Comment:
Since this is tags plural, we need a way to separate the tags - I assume commas based on the kitchen sink PR. That separator needs to be added to the regex I think.

Reply:
Good spot, yes it's a comma without spaces to delimit